lib/java-javadoc.gradle

```diff
@@ -174,6 +174,39 @@ allprojects {
                 return success
             }
 
+            def addJdk11OfflineLink = { name, url, elementListFile ->
+                def modulePrefix = 'module:';
+                def inModule = false
+                def currentModule = ''
+                List<String> lines = []
+                elementListFile.eachLine { line ->
+                    if (line.startsWith(modulePrefix)) {
+                        if (inModule) {
```

Comment:
I'm not sure but is `inModule` flag unnecessary? I think it can be replaced with checking `currentModule` is not blank.

Reply:
Yeah, I can remove it. Thanks~!